plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.3.1)
+
+FetchContent_MakeAvailable(Corrosion)
+
+set(obs-webrtc_HEADERS whip-output.h)
+set(obs-webrtc_SOURCES whip-service.c whip-output.c webrtc.c)
+
+if(WIN32)
+  set(MODULE_DESCRIPTION "OBS webrtc module")
+  configure_file(${CMAKE_SOURCE_DIR}/cmake/bundle/windows/obs-module.rc.in
+                 obs-webrtc.rc)
+  list(APPEND obs-webrtc_SOURCES obs-webrtc.rc)
+endif()
+
+add_library(obs-webrtc MODULE ${obs-webrtc_SOURCES} ${obs-webrtc_HEADERS})
```

Comment:
Always define the target at the top of the file, then use `target_*` functions to add sources after. Avoid using variables with file lists (that’s CMake 2.x style, we require CMake 3.16 style at least).

Reply:
Fixed d7f7e67